### PR TITLE
Add PLAYA3ULL Games Mainnet

### PR DESCRIPTION
**Pull request by samthompsonkennedy**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
https://explorer.playa3ull.games cannot be reached

**Comment by samthompsonkennedy** [comment]
@ligi This one has been resolved, updated to the online explorer.

File: _data/chains/eip155-3011.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "shortName": "3ULL",
+  "chainId": 3011,
+  "networkId": 3011,
+  "icon": "playa3ull",
+  "explorers": [
+    {
+      "name": "PLAYA3ULL GAMES Explorer",
+      "url": "https://avascan.info/blockchain/playa3ull/home",
+      "icon": "playa3ull",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-3011.json:22]: The explorer is not EIP3091 compatible
```suggestion
      "standard": "none"
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-3011.json (hunk 1 above)
Comment (samthompsonkennedy) [reply _data/chains/eip155-3011.json:22]: @ligi changed to routerscan which is etherscan compliant.

File: _data/chains/eip155-3011.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "features": [{ "name": "EIP1559" }],
+  "infoURL": "https://playa3ull.games",
+  "shortName": "3ULL",
+  "chainId": 3011,
+  "networkId": 3011,
+  "icon": "playa3ull",
+  "explorers": [
+    {
+      "name": "PLAYA3ULL GAMES Explorer",
+      "url": "https://3011.routescan.io/",
```
Comment (ligi) [inline _data/chains/eip155-3011.json:20]:
```suggestion
      "url": "https://3011.routescan.io",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Pull request merged**